### ISSUE TITLE
Suggestion about web app and app version of Morphic one day

### ISSUE THREAD

**Issue by ZainGithub12** [miurla/morphic#296]
@miurla, When there's an app version of Morphic on app store and playstore, I think it would be good to keep the web version as well for flexibility. So having both web and app version. Was this your plan anyway? 🙂

**Comment by miurla** [comment]
I agree. However, it depends on the team's resources, so it’s undecided at this stage.
